### PR TITLE
Add privacy policy jaggery app

### PR DESCRIPTION
**Pull request by lasanthaS**
## Purpose
This PR adds privacy policy jaggery app to comply with GDPR.

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: modules/distribution/src/privacy-policies/jaggery.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    {
+      "url":"/privacy-policy",
+      "path":"/pages/privacy-policy.jag"
+    },
+    {
+      "url":"/cookie-policy",
+      "path":"/pages/cookie-policy.jag"
+    }
+  ]
+}
```
Comment (this) [inline modules/distribution/src/privacy-policies/jaggery.conf]: Please add file-ending new line.

File: modules/distribution/src/privacy-policies/pages/cookie-policy.jag [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    </div>
+</footer>
+
+<script src="libs/jquery_1.11.3/jquery-1.11.3.js"></script>
+<script src="libs/bootstrap_3.3.5/js/bootstrap.min.js"></script>
+
+<script type="text/javascript" src="js/u2f-api.js"></script>
+
+</body>
+</html>
```
Comment (this) [inline modules/distribution/src/privacy-policies/pages/cookie-policy.jag]: Please add file-ending new line.

File: modules/distribution/src/privacy-policies/pages/privacy-policy.jag [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    </div>
+</footer>
+
+<script src="libs/jquery_1.11.3/jquery-1.11.3.js"></script>
+<script src="libs/bootstrap_3.3.5/js/bootstrap.min.js"></script>
+
+<script type="text/javascript" src="js/u2f-api.js"></script>
+
+</body>
+</html>
```
Comment (this) [inline modules/distribution/src/privacy-policies/pages/privacy-policy.jag]: Please add file-ending new line.

Comment (lasanthaS) [reply modules/distribution/src/privacy-policies/pages/privacy-policy.jag]: Done

File: modules/distribution/src/privacy-policies/pages/cookie-policy.jag (hunk 2 above)
Comment (lasanthaS) [reply modules/distribution/src/privacy-policies/pages/cookie-policy.jag]: Done

File: modules/distribution/src/privacy-policies/jaggery.conf (hunk 1 above)
Comment (lasanthaS) [reply modules/distribution/src/privacy-policies/jaggery.conf]: Done

**Pull request merged**